src/app.ts

```diff
@@ -157,19 +171,31 @@ app.on('ready', () => {
     ]
 
     if (is.linux) {
-      contextMenuTemplate.unshift({
-        click: () => {
-          mainWindow.show()
+      contextMenuTemplate.unshift(
```

Comment:
Could we keep this as a single item instead of two separate items?  Or would that get too messy?

Reply:
That's a good idea indeed.

Reply:
> Correct. More like `Start minimized` maybe?

@timche, I'm trying to understand: "Start minimized" can mean different things, for instance, when "Start minimized" is enabled and the user tries to launch Gmail from Launchpad, it could start silently in the tray, And the another meaning could be that "Start Minimized" will actually auto-start Gmail at system startup minimized to the tray. Which of this did you mean?